### PR TITLE
Icon Added

### PR DESCRIPTION
**Pull request by Slothjams**
unexpected icon: 0156_questionMark_i_diamond_light.svg

### CODE REVIEW

File: src/CernerOneIcons-CSV.csv [hunk 1]
```diff
@@ -152,3 +152,4 @@ reload,src/icons/0152_circleArrow_a.svg,themeable ,
 recurringEvent,src/icons/0153_calendarCircleArrow_a.svg,themeable ,
 separate,src/icons/0154_lineTwoArrows_a.svg,themeable ,bi-directional
 merge,src/icons/0155_connectedLinesArrow_a.svg,themeable ,bi-directional
+unexpected,src/icons/0156_questionMark_i_diamond_light.svg,,
```
Comment (Matt-Butler) [inline src/CernerOneIcons-CSV.csv:155]: Will this svg be a static color?

Comment (sethbroox) [reply src/CernerOneIcons-CSV.csv:155]: yes

**Comment by sethbroox** [comment]
+1

File: src/CernerOneIcons-CSV.csv (hunk 1 above)
Comment (Matt-Butler) [reply src/CernerOneIcons-CSV.csv:155]: cool, thanks

**Pull request merged**